libs/muddle/src/peer_selector.cpp

```diff
@@ -32,18 +32,21 @@
 namespace fetch {
 namespace muddle {
 
+namespace {
+
 using namespace std::chrono;
 using namespace std::chrono_literals;
 
-static constexpr auto        MIN_ANNONCEMENT_INTERVAL = 15min;
-static constexpr auto        MAX_ANNONCEMENT_INTERVAL = 30min;
-static constexpr std::size_t MINIMUM_PEERS            = 3;
-static constexpr char const *BASE_NAME                = "PeerSelector";
-static constexpr std::size_t MAX_CACHE_KAD_NODES      = 20;
-static constexpr std::size_t MAX_CONNECTED_KAD_NODES  = 3;
+constexpr auto        MIN_ANNONCEMENT_INTERVAL = 15min;
+constexpr auto        MAX_ANNONCEMENT_INTERVAL = 30min;
+constexpr std::size_t MINIMUM_PEERS            = 3;
+constexpr char const *BASE_NAME                = "PeerSelector";
+constexpr std::size_t MAX_CACHE_KAD_NODES      = 20;
+constexpr std::size_t MAX_CONNECTED_KAD_NODES  = 3;
+constexpr std::size_t MAX_LOG2_BACKOFF         = 11;  // 2048
 
-static std::unordered_set<Address> operator+(std::unordered_set<Address>        input,
-                                             std::unordered_set<Address> const &other)
+std::unordered_set<Address> operator+(std::unordered_set<Address>        input,
```

Comment:
Shouldn't this be turned into a library function? Looks too fundamental/simple/generic for an anonymous namespace.

Reply:
Done